src/bot.py

```diff
@@ -75,40 +123,43 @@ def tweet_loop(self, api):
                 else:
                     api.update_status(self.generate_batch())
                     logging.info("Tweeted out a new flower bed!")
-                    logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
+                    logging.info("The next tweet is scheduled to be made in {} seconds".format(cooldown))
                     time.sleep(cooldown)
-            except:
-                print(sys.exc_info())
-                logging.critical("Tweeting failed due to ratelimit. Waiting {} more minutes.".format(cooldown))
-                time.sleep(cooldown)
+                    raise tweepy.TweepError("error")
+            except Exception as ex:
+                exception = type(ex).__name__
                 
+                if exception == 'RateLimitError':
+                    logging.critical("Tweeting failed due to ratelimit. Waiting {} more seconds.".format(cooldown))
+                    time.sleep(cooldown)
+                else:
+                    logging.critical("TweepError not RateLimitError.")
+                    
     def main(self):
         '''
         Main function that takes care of the authentication and the threading.
         '''
+        
         logging.info("Attempting to login!")
-
-
-        # Authenticate to Twitter
         auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
         auth.set_access_token(ACCESS_KEY, ACCESS_SECRET)
-
         api = tweepy.API(auth)
-
-        try:
-            api.verify_credentials()
-            logging.info("Successfully logged in!")
-            self.tweet_loop(api)
-        except:
-            print("Error during authentication")
-
+        user = api.me()
+        self.user_info(api, user)
+        logging.info("Successfully logged in!")
+        self.tweet_loop(api)
 
 if __name__ == '__main__':
     try:
-        
+        FLAG = True
         TwitterBot = Bot()
-        

```

Comment:
How are the line deletions relevant to the issue? Again, this is abandoned code, I'd be glad to discuss this, but why do that here?